ztools/xssh/kex.go

```diff
@@ -165,7 +165,7 @@ type dhGroup struct {
 	JsonLog       dhGroupJsonLog
 }
 type dhGroupJsonLog struct {
-	Parameters      *ztoolsKeys.DHParams  `json:"params,omitempty"`
+	Parameters      *ztoolsKeys.DHParams  `json:"dh_params,omitempty"`
```

Comment:
Alright. We should file an issue noting that all of this is confusing, and a little bit about how it works, and note that structure should probably be reconsidered in some way.